### ISSUE TITLE
Ubuntu and Debian packages in OBS are missing libddcutil.so.x.x.x

### ISSUE THREAD

**Issue by w8jcik** [rockowitz/ddcutil#425]
Ubuntu and Debian packages in OpenSUSE Build Service are missing `libddcutil.so`. They only contain a broken symlink. Not sure about the other distributions.

# Details

Example Docker recipe to reproduce the issue

```Dockerfile
FROM ubuntu:24.04

ARG SUSE_NAME="xUbuntu_24.04"
ARG DEBIAN_FRONTEND=noninteractive

RUN apt update -y
RUN apt upgrade -y

RUN apt install libglib2.0-dev -y
RUN apt install curl gpg -y

RUN echo "deb http://download.opensuse.org/repositories/home:/rockowitz/${SUSE_NAME}/ /" | tee /etc/apt/sources.list.d/home:rockowitz.list
RUN curl -fsSL "https://download.opensuse.org/repositories/home:rockowitz/${SUSE_NAME}/Release.key" | gpg --dearmor | tee /etc/apt/trusted.gpg.d/home_rockowitz.gpg > /dev/null

RUN apt update -y
RUN apt install ddcutil libddcutil-dev -y
```

Listing files in a package `dpkg -L libddcutil-dev`

```
/.
/usr
/usr/include
/usr/include/ddcutil_c_api.h
/usr/include/ddcutil_macros.h
/usr/include/ddcutil_status_codes.h
/usr/include/ddcutil_types.h
/usr/lib
/usr/lib/x86_64-linux-gnu
/usr/lib/x86_64-linux-gnu/cmake
/usr/lib/x86_64-linux-gnu/cmake/ddcutil
/usr/lib/x86_64-linux-gnu/cmake/ddcutil/FindDDCUtil.cmake
/usr/lib/x86_64-linux-gnu/pkgconfig
/usr/lib/x86_64-linux-gnu/pkgconfig/ddcutil.pc
/usr/share
/usr/share/doc
/usr/share/doc/libddcutil-dev
/usr/share/doc/libddcutil-dev/AUTHORS
/usr/share/doc/libddcutil-dev/CHANGELOG.md.gz
/usr/share/doc/libddcutil-dev/NEWS.md.gz
/usr/share/doc/libddcutil-dev/changelog.Debian.gz
/usr/share/doc/libddcutil-dev/copyright
/usr/lib/x86_64-linux-gnu/libddcutil.so
```

Listing files in a package `dpkg -L libddcutil4`

```
/.
/usr
/usr/share
/usr/share/doc
/usr/share/doc/libddcutil4
/usr/share/doc/libddcutil4/changelog.Debian.gz
/usr/share/doc/libddcutil4/changelog.gz
/usr/share/doc/libddcutil4/copyright
```

Inspecting `/usr/lib/x86_64-linux-gnu/libddcutil.so`

```
ls /usr/lib/x86_64-linux-gnu -l
```

```
...
lrwxrwxrwx  1 root root       19 Feb  7 09:00 libddcutil.so -> libddcutil.so.5.1.2 
...
```

(broken link, `/usr/lib/x86_64-linux-gnu/libddcutil.so.5.1.2` does not exist)

I noticed it trying to build `ddcutil-service`

```bash
make
```

```
...
/usr/bin/ld: cannot find -lddcutil: No such file or directory
...
```

The same issue is present in the other Ubuntu and Debian versions.

Are OBS recipes available in some public repository? It would help diagnosing and fixing packaging issues.

**Comment by w8jcik** [comment]
Sorry for closing and reopening the issue. I thought I found the solution, but I was wrong. The issue is still here.

**Comment by w8jcik** [comment]
Thank you for fixing it. The library is present now and it works fine. The issue can be closed.

**Comment by rockowitz** [comment]
Fixed flibddcutil5-2.1.3-1  and libddcutil5-2.1.4-1 for Debian and Ubuntu releases.   For RPM based distributions (e.g. openSUSE, Fedora), the packaging is orchestrated by file ddcutil.spec.  For Debian based distributions, build is orchestrated by the files in directory debian of packages ddcutil_2.1.4-1.debian.tar.xz, etc.
